### PR TITLE
pcsx2 > Speedhacks & Game Fixes - GUI Textstring Alterations

### PR DESCRIPTION
**Pull request by pgert**
Changes translatable strings.

### CODE REVIEW

**Comment by pgert** [comment]
Any support for this?

**Comment by lightningterror** [comment]
Are these changes really necessary ?
Imo the original look better.

File: pcsx2/gui/Panels/SpeedhacksPanel.cpp [hunk 1]
```diff
@@ -115,10 +115,10 @@ Panels::SpeedHacksPanel::SpeedHacksPanel( wxWindow* parent )
 	const wxSizerFlags sliderFlags( wxSizerFlags().Border( wxLEFT | wxRIGHT, 8 ).Expand() );
 
 	m_check_Enable = new pxCheckBox( this, _("Enable speedhacks"),
-		pxE( L"Speedhacks usually improve emulation speed, but can cause glitches, broken audio, and false FPS readings.  When having emulation problems, disable this panel first."
+		pxE( L"Speedhacks overall increases emulation speed, but can cause glitches, broken audio, and/or false FPS readings.\nWhen having emulation problems, disable (or alter) this panel first, in troubleshooting."
```
Comment (lightningterror) [inline pcsx2/gui/Panels/SpeedhacksPanel.cpp]: *increase

**Comment by bositman** [comment]
Original has easier english, I don't think you will improve user experience with more complex wording

**Comment by ssakash** [comment]
I don't see how this is better than the current description.

File: pcsx2/gui/Panels/SpeedhacksPanel.cpp (hunk 1 above)
Comment (refractionpcsx2) [reply pcsx2/gui/Panels/SpeedhacksPanel.cpp]: "overall increase emulation speed" isn't always true, hence the original wording.
Also I'd rather people disabled speedhacks when having problems, if you just alter them, you may notice no difference and then assume speedhacks aren't the problem.

**Comment by MrCK1** [comment]
These changes would also require a revision of other translations, I don't see the point.

**Comment by pgert** [comment]
Well thanx for the feedback.
I'll close the PR in a couple of days then.
I seemed better to me ...

**Pull request closed without merging**